packaging/overlays/strip-status.yaml

```diff
@@ -1,20 +0,0 @@
-# Copyright 2021 VMware
```

Comment:
don't we still need this?

Reply:
ah, I see it's in the package repo https://github.com/vmware-tanzu/package-for-cartographer/blob/main/src/cartographer/config/overlays/strip-status.yaml